homeassistant/components/cloud/auth_api.py

```diff
@@ -0,0 +1,264 @@
+"""Package to offer tools to authenticate with the cloud."""
+import json
+import logging
+import os
+
+from .const import AUTH_FILE, SERVERS
+from .util import get_mode
+
+_LOGGER = logging.getLogger(__name__)
+
+
+class CloudError(Exception):
+    """Base class for cloud related errors."""
```

Comment:
I think that should be inside `__init__`

Reply:
These are the class docstrings. There is no need to implement `__init__` because we want to keep the base class constructor.